content/nuxt.md

```diff
@@ -3,4 +3,265 @@ path: "/docs/nuxt"
+
+Create a file called `nuxt.config.js` in `guess-nuxt` with the following content:
+
+```javascript
+import { readFileSync } from 'fs'
+import { GuessPlugin } from 'guess-webpack'
+
+export default {
+  build: {
+    extend(config, ctx) {
+      if (ctx.isClient) {
+        config.plugins.push(
+          new GuessPlugin({
+            reportProvider() {
+              return Promise.resolve(JSON.parse(readFileSync('./routes.json')))
+            }
+          })
+        )
+      }
+    }
+  }
+}
+```
+
+Now create a file called `routes.json`, in the same directory as `nuxt.config.js`, and add the following content:
+
+```json
+{
+  "/": {
+    "/example": 80,
+    "/about": 20
+  },
+  "/example": {
+    "/": 20,
+    "/media": 0,
+    "/about": 80
+  },
+  "/about": {
+    "/": 20,
+    "/media": 80
+  },
+  "/media": {
+    "/": 33,
+    "/about": 33,
+    "/example": 34
+  }
+}
+```
+
+This file specifies how many times the user has visited a given page from another. For example, if we look at the first property of the outermost object, we can see that from `/`, there were `80` sessions in which users have visited `/example` and `20` sessions in which users have visited `/about`.
+
+> Based on the content of this file, Guess.js is going to build a model for predictive prefetching.
+
+<div style="background-color: #e7f4ff; border-radius: 5px; padding: 20px; margin-bottom: 20px;">
+Alternatively, if you're using Google Analytics, instead of providing a <code>reportProvider</code>, you can set the <code>GA</code> property with value your Google Analytics View ID. In this case, Guess.js will fetch data from your Google Analytics account and build the report automatically. For the purpose, you'll have to provide a read-only access to your view.
+</div>
+
+In this particular case there is slight changes in `nuxt.config.js`:
+
+```javascript
+import { readFileSync } from 'fs'
+import { GuessPlugin } from 'guess-webpack'
+
+const { GA } = process.env
+
+export default {
+  build: {
+    extend(config, ctx) {
+      if (ctx.isClient) {
+        const guessOptions = {
+          runtime: {
+            delegate: true,
+            prefetchConfig: {
+              '4g': 0.3,
+              '3g': 0.3,
+              '2g': 0.3,
+              'slow-2g': 0.3
+            }
+          },
+          routeProvider: false
+        }
+        if (GA) guessOptions.GA = GA
+        else guessOptions.reportProvider = () => Promise.resolve(JSON.parse(readFileSync('./routes.json')))
+
+        config.plugins.push(
+          new GuessPlugin(guessOptions)
+        )
+      }
+    }
+  }
+}
+```
+
+Alternative for alternatively, a  [@nuxtjs/guess](https://github.com/daliborgogic/guess-module) module for Guess.js!
+
+### Setup
+
+- Add `@nuxtjs/guess` dependency to your project
+- Add `@nuxtjs/guess` to modules section of `nuxt.config.js`
+
+```json
+{
+  modules: [
```

Comment:
GitHub complains because the property names & values are not within double quotes.